docs/updating-components.md

```diff
@@ -0,0 +1,74 @@
+# How to update components?
+
+This document lists the steps to update components in Lokoctl.
+
+## Calico
+
+To update calico update the image tags in following files:
```

Comment:
```suggestion
To update Calico, update the image tags in following files:
```

@surajssd we do our best to capitalize project names using the official project's style (Calico != calico). Could you please fix that in the rest of the PR?